pallets/bridge/src/lib.rs

```diff
@@ -230,6 +231,11 @@ pub mod pallet {
     #[pallet::getter(fn chain_nonces)]
     pub type ChainNonces<T: Config> = StorageMap<_, Blake2_256, ChainId, ChainNonce, ValueQuery>;
 
+    #[pallet::storage]
+    #[pallet::getter(fn bridge_registry)]
+    pub type BridgeRegistry<T: Config> =
+        StorageMap<_, Blake2_128Concat, ChainId, Vec<BridgeId>, OptionQuery>;
```

Comment:
shall we use `BoundedVec` instead?

Reply:
Related:
- https://github.com/parallel-finance/parallel/issues/1272#issue-1129688982